run_it_simple_PT.jl

```diff
@@ -0,0 +1,151 @@
+using Printf, LinearAlgebra, Statistics, Plots
+
+@views   inn(A) = A[2:end-1,2:end-1]
+@views av_xa(A) = (0.5  .* (A[1:end-1,:] .+ A[2:end,:]))
+@views av_ya(A) = (0.5  .* (A[:,1:end-1] .+ A[:,2:end]))
+@views av_xi(A) = (0.5  .* (A[1:end-1,2:end-1] .+ A[2:end,2:end-1]))
+@views av_yi(A) = (0.5  .* (A[2:end-1,1:end-1] .+ A[2:end-1,2:end]))
+@views    av(A) = (0.25 .* (A[1:end-1,1:end-1] .+ A[1:end-1,2:end] .+ A[2:end,2:end] .+ A[2:end,1:end-1]))
+
+const small = eps(Float64)
+const day   = 24*3600
+
+@views function simple_sheet(; do_monit=true)
+    # physics
+    Lx, Ly = 100e3, 20e3                  # length/width of the domain, starts at (0, 0)
+    dt     = 1e7                          # physical time step
+    α      = 1.25
+    β      = 1.5
+    m      = 7.93e-11                           # source term for SHMIP A1 test case
+    e_v    = 0.1                               # void ratio for englacial storage
+
+    # numerics
+    nx, ny = 64, 32
+    nout   = 1000
+    itMax  = 10^4
+    γ_h    = 0.8
+    γ_ϕ    = 0.8
+
+    # derived
+    dx, dy = Lx / (nx-3), Ly / (ny-3)     # the outermost points are ghost points
+    xc, yc = LinRange(-dx, Lx+dx, nx), LinRange(-dy, Ly+dy, ny)
+
+    # initial conditions
+    ϕ0     = 100. * ones(nx, ny)
+    h0     = 0.04 * ones(nx, ny) # initial fields of ϕ and h
+    get_H(x, y) = 6 *( sqrt((x)+5e3) - sqrt(5e3) ) + 1
+    H      = [0.0; ones(nx-2); 0.0] * [0.0 ones(ny-2)' 0.0] .* get_H.(xc, yc') # ice thickness, rectangular ice sheet with ghostpoints
+    ϕ0[1:2,:] .= 0.0 # Dirichlet BC
+
+    # scaling factors
+    H_     = 1000.0
+    ϕ_     = 9.81 * 910 * mean(H)
```

Comment:
Here `H_`

Reply:
Thanks! Fixed in b3177c1c3f408e9e5cb1dec58a94c7bb1288b839 and since the error was already in the main branch 1c900c33bc68e4b8640276413900e11130bc7bed.